docs/tutorials/t2ramsey_tutorial.ipynb

```diff
@@ -0,0 +1,416 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "backend_in_ns = T2RamseyBackend(\n",
+    "                    p0={\"a_guess\":[0.5], \"t2ramsey\":[80000], \"f_guess\":[0.00002], \"phi_guess\":[0.0],\n",
+    "                        \"b_guess\": [0.5]},\n",
+    "                    initial_prob_plus=[0.0],\n",
+    "                    readout0to1=[0.02],\n",
+    "                    readout1to0=[0.02],\n",
+    "                    conversion_factor=conversion_factor\n",
+    "                )\n",
+    "exp_in_ns = T2Ramsey(qubit, delays, unit=unit)\n",
+    "exp_in_ns.set_analysis_options(user_p0=None, plot=True)\n",
+    "expdata_in_ns = exp_in_ns.run(backend=backend_in_ns, shots=2000)\n",
+    "print(expdata_in_ns)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is possible to add data to an experiment, after the analysis of the first set of data. In the next example we add exp2 to `exp_in_ns` that we showed above."
```

Comment:
Add subtitle, something like "Adding data to an existing experiment"

Reply:
ok, done.